### PR TITLE
feat: Budget notification improvements

### PR DESCRIPTION
**Pull request by ptbrowne**
- Account and group labels are shown
- Push notification content
- Email footer is fixed
- Instead of launching function for category budgets, we launch a job for better monitoring in metabase/bender

### CODE REVIEW

File: src/ducks/budgetAlerts/CategoryBudgetNotificationView.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const transformForTemplate = (budgetAlert, t, accountsById, groupsById) => {
   return {
     ...budgetAlert.alert,
     categoryLabel: t(`Data.${type}.${catName}`),
-    currentAmount: (-sumBy(budgetAlert.expenses, tr => tr.amount)).toFixed(2),
+    currentAmount: (-sumBy(budgetAlert.expenses, tr => tr.amount)).toFixed(0),
```
Comment (y-lohse) [inline src/ducks/budgetAlerts/CategoryBudgetNotificationView.js]: Nit: is there a point of using `toFixed(0)`, as opposed to flooring or rounding the number?

File: src/targets/services/onOperationOrBillCreate.js [hunk 2]
```diff
@@ -158,7 +158,12 @@ const onOperationOrBillCreate = async options => {
   setting = await updateSettings(setting)
 
   log('info', 'Do category budgets notification')
-  await runCategoryBudgetService(cozyClient)
+  const client = CozyClient.fromEnv(process.env)
+  const jobs = client.collection('io.cozy.jobs')
+  await jobs.create('service', {
+    name: 'budgetAlerts',
+    slug: 'banks'
+  })
```
Comment (y-lohse) [inline src/targets/services/onOperationOrBillCreate.js]: Out of curiosity, why the change from service to job?

Comment (ptbrowne) [reply src/targets/services/onOperationOrBillCreate.js]: "from function to job" I guess. Because this enables finer grained monitoring in Bender and in metabase.

File: src/ducks/budgetAlerts/CategoryBudgetNotificationView.js (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/budgetAlerts/CategoryBudgetNotificationView.js]: Not really, I had toFixed(2) before and I changed it to toFixed(0) ;)
